static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,333 @@
         }
     }
 }
+
+// New gallery stuff
+
+// TODO: put in pasteup
+$multimedia-support-5: #161616;
+
+// The main image needs to take up 100vh and the slim header needs to sit just on top of it.
+.l-header--new-gallery {
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    left: 0;
+}
```

Comment:
Note: Olly has come up with a fix for this and is implementing it on a bigger PR as it will affect other things as well